todo_app/app.py

```diff
@@ -1,15 +1,46 @@
-from flask import Flask
+from flask import render_template, Flask, request, redirect, url_for
+from todo_app.data.session_items import *
```

Comment:
(Minor Point) Generally it's a good practice not to use global imports (i.e. the *). We'd recommend deglob-ing them.